Source/ConvolutionFunctions/arm_convolve_get_buffer_sizes_s8.c

```diff
@@ -65,13 +65,13 @@ __STATIC_INLINE int32_t arm_convolve_1x1_s8_fast_get_buffer_size_mve(const cmsis
     const int32_t lhs_rows = input_dims->w * input_dims->h * input_dims->n;
     const int32_t rhs_rows = filter_dims->n;
 
-    if (lhs_rows % 3 == 0 && rhs_rows % 2 == 0)
+    if (lhs_rows % 3 == 0 && rhs_rows % 2 == 0 && rhs_rows > lhs_rows * 4)
```

Comment:
What is the intention behind rhs_rows > lhs_rows * 4 ?

Reply:
It is work in progress for running the new mat_mult operator only on cases where it improves performance, removing for this bug fix.